src/chapters/databases-part2/home-logic.rst

```diff
@@ -0,0 +1,60 @@
+Home Page Logic
+===============
+
+Now that we have our HTML templates ready, it's time to add some Python to get
+the game running.
+
+The ``index()`` Function
+------------------------
+
+The home page for our application is controlled by the ``index()`` function in
+``main.py``. It serves several purposes:
+
+#. To clear existing data in preparation for a new game,
+#. To process the ``Main Menu`` form submission,
+#. To hide a new batch of mines on the board,
+#. To redirect a player to the URL where they will play the game.
+
+To complete your ``index()`` function, open your Minesweeper project and code
+along with the video:
+
+.. raw:: html
+
+   <section class="vid_box">
+      <iframe class="vid" src="https://www.youtube-nocookie.com/embed/OZ1-PDXDZwI" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+
```

Comment:
at 1.22min in the video:

def record_mines(locations) = def record_mines(coords) in starter code

Reply:
Good catch!  I fixed the starter code to match.